tests/ui/macros/issue-88228.stderr

```diff
@@ -5,7 +5,7 @@ LL |     bla!();
    |     ^^^
    |
    = help: consider importing this macro:
-           crate::hey::bla
+           use crate::hey::bla;
```

Comment:
Is there any way that we can make this into a [`suggestion`](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_errors/diagnostic/struct.Diagnostic.html#method.span_suggestions) instead of just changing this help message?

Reply:
```rust
   err.span_suggestion(
      expr.span,
      "give it a value of the expected type",
       format!("break{label} {val}"),
       Applicability::HasPlaceholders,
     );
 ```
 
 We would need to pass in the span as the first argument and I am wondering what exactly should be passed in there because span goes out of scope before the else block.